### PR TITLE
odc-5564-select the builder image when git url is not validated

### PR DESCRIPTION
**Pull request by makambalaji**
**problem**: "git url is not validated due to git rate limit
**solution**: Enter the git url, if it is not validated, then select the builder image manually

**JIRA Id**:  https://issues.redhat.com/browse/ODC-5564

image

image

image

### CODE REVIEW

File: frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/create-from-add-options.ts
Comment (VeereshAradhya) [inline frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/create-from-add-options.ts:84]: If the implementation is removed the step from feature file also should be removed right?

Comment (makambalaji) [reply frontend/packages/pipelines-plugin/integration-tests/support/step-definitions/pipelines/create-from-add-options.ts:84]: Its duplicate one, so thought to remove it

File: frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts
Comment (VeereshAradhya) [inline frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts:133]: the `builderImageName` will have default value of `Node.js`. What if another URL is given? The step should include the `builderImageName` as param. WDYT?

File: frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts
Comment (VeereshAradhya) [inline frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts:142]: how about logging `selecting builder image builderImageName` in case of fail in validation?

File: frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts:133]: Even if we give different url, still the script works fine, because builder image selection doesn't impact on any screen

File: frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts:142]: Added

File: frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts
Comment (VeereshAradhya) [reply frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts:133]: If the validation fails, how the script will know what builder image (nodejs, python, php, etc..) it has to select?

**Comment by makambalaji** [comment]
Holding until #8289 merges to unblock CI.

/hold

File: frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/support/pages/add-flow/git-page.ts:133]: 1st point - 
We already used param, but the param got default value 'Node.js'. [If we are not passing any parameter. it autoselects the node.js builder image by default]
3rd point -
Based on the scenario, we can identify the test data.

**Comment by makambalaji** [comment]
PR: #8289  got merged
/hold cancel

**Review by gajanan-more — APPROVED** [review]
Running as expected.
/lgtm

**Pull request merged**
